lib/rollbar/plugins/delayed_job/plugin.rb

```diff
@@ -53,7 +53,10 @@ def self.report(e, job)
     end
 
     def self.skip_report?(job)
-      job.attempts < ::Rollbar.configuration.dj_threshold
+      job.attempts < ::Rollbar.configuration.dj_threshold && (
```

Comment:
The name `only_report_failures` doesn't make sense to me. The only case that is different is when the number of attempts is less than the threshold, only_report_failures is true and the job has failed. Before this change that would have been a skip, and with this change that would be a report. So we are actually reporting failures under the threshold, but we are still reporting any time the number of attempts is greater or equal to the threshold regardless of whether the job failed or not.